packages/circuit-ui/components/Popover/Popover.stories.tsx

```diff
@@ -55,16 +56,26 @@ const actions = [
   },
 ];
 
-export const Base = (args: PopoverProps): JSX.Element => (
-  <Popover
-    {...args}
-    component={forwardRef((props, ref) => (
-      <Button size="kilo" variant="primary" ref={ref} {...props}>
-        Open popover
-      </Button>
-    ))}
-  />
-);
+export const Base = (args: PopoverProps): JSXElement => {
```

Comment:
That's not the right type. `JSX` is a global type provided by TypeScript. Make sure to also remove the import on line 20.

```suggestion
export const Base = (args: PopoverProps): JSX.Element => {
```